plugins/woocommerce/includes/admin/class-wc-admin-notices.php

```diff
@@ -116,8 +120,42 @@ public static function reset_admin_notices() {
 		self::add_notice( 'template_files' );
 		self::add_min_version_notice();
 		self::add_maxmind_missing_license_key_notice();
+		self::maybe_add_php73_required_notice();
+	}
+
+	// phpcs:disable Generic.Commenting.Todo.TaskFound
+
+	/**
+	 * Add an admin notice about the bump of the required PHP version in WooCommerce 7.7
+	 * if the current PHP version is too old.
+	 *
+	 * TODO: Remove this method in WooCommerce 7.7.
+	 */
+	private static function maybe_add_php73_required_notice() {
+		if ( version_compare( phpversion(), '7.3', '>=' ) ) {
+			return;
+		}
+
+		self::add_custom_notice(
+			'php73_required_in_woo_77',
+			__( '<h4>PHP version requirements will change soon</h4><p>WooCommerce 7.7, scheduled for <b>May 2023</b>, will require PHP 7.3 or newer to work. Your server is currently running an older version of PHP, so this change will impact your store. Upgrading to at least PHP 8.0 is recommended. <b><a href="https://developer.woocommerce.com/2023/01/10/new-requirement-for-woocommerce-7-7-php-7-3/">Learn more about this change.</a></b></p>', 'woocommerce' )
```

Comment:
Probably not feasible to remove _all_ the HTML from the translatable string, but some of it could be removed, which would probably improve UX for translators and avoid potential errors. One possible approach:

```suggestion
            sprintf(
                '%s%s',
                sprintf(
                    '<h4>%s</h4>',
                    esc_html__( 'PHP version requirements will change soon', 'woocommerce' )
                ),
                sprintf(
                    wpautop( wp_kses_data( __( 'WooCommerce 7.7, scheduled for <b>May 2023</b>, will require PHP 7.3 or newer to work. Your server is currently running an older version of PHP, so this change will impact your store. Upgrading to at least PHP 8.0 is recommended. <b><a href="%s">Learn more about this change.</a></b>', 'woocommerce' ) ) ),
                    'https://developer.woocommerce.com/2023/01/10/new-requirement-for-woocommerce-7-7-php-7-3/'
                )
            );
```

Reply:
This notice is temporary and the code changes are pretty much copied verbatim from https://github.com/woocommerce/woocommerce/pull/31557 so I'd normally say that it's not worth the effort, but since you have provided a ready to use alternative, why not using it? 😄 Added in a separate commit (didn't use "Commit suggestion" because there was an extra semicolon).